src/Components/Shared/Modal/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react';
+import styles from './modal.module.css';
+
+/**This component is used by passing the properties of title for the title, desc for the description, isOpen to open the modal, confirmBtn for the confirm or accept button, denyBtn for the cancel or exit button, clickFunction, which is what suits it to indicate to the confirm button the action you want to perform and chooseModal to determine whether to use a modal that requires a confirmation or a modal that shows a text that disappears after 5 seconds */
+
+const Modal = ({ title, desc, isOpen, confirmBtn, denyBtn, chooseModal, close, clickFunction }) => {
```

Comment:
Here you can use onClick so it's easiest to read later when we use the component

```suggestion
const Modal = ({ title, desc, isOpen, confirmBtn, denyBtn, chooseModal, close, onClick }) => {
```